modules/prebidServerBidAdapter/index.js

```diff
@@ -849,6 +849,11 @@ Object.assign(ORTB2.prototype, {
       }
     };
 
+    // If the price floors module is active, then we need to signal to PBS! If floorData obj is present is best way to check
+    if (typeof deepAccess(firstBidRequest, 'bids.0.floorData') === 'object') {
+      request.ext.prebid.floors = { enabled: false };
```

Comment:
if floor data exists, wouldnt enabled be true? does this mean deactivate the serverside floor bc a client side floor was chosen?

Reply:
I too thought about that haha

But I think this is a signal saying "Do not run Price Floors logic on PBS for this Auction"

@bretg to confirm!

Reply:
right - ext.prebid.floors.enabled:false will turn off the server-side floors feature, causing it to default to imp.bidfloor.